lib/capture_migration_sql.rb

```diff
@@ -12,10 +14,10 @@ class << self
     # The `starting_with` argument can be used to specify which migration you
     # wish to start capturing SQL with. This can be useful if you are adding
     # this gem to an existing project with a history of migrations that you
-    # don't want to go back and edit.
+    # don't want to go back an edit.
```

Comment:
Kinda understand what you're saying, but the phrasing seems really weird. Definitely _feels_ like a typo.